package.json

```diff
@@ -40,14 +41,15 @@
   },
   "devDependencies": {
     "eslint": "^4.3.0",
-    "prettier": "^1.1.0",
-    "prettier-eslint": "^6.4.2",
     "eslint-config-airbnb": "^15.1.0",
     "eslint-config-prettier": "^2.3.0",
     "eslint-plugin-flowtype": "^2.30.3",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^5.1.1",
     "eslint-plugin-prettier": "^2.0.1",
-    "eslint-plugin-react": "^7.1.0"
+    "eslint-plugin-react": "^7.1.0",
+    "husky": "^0.14.3",
```

Comment:
this will add also to the generated project? I thought this was for the bootstrap itself

Reply:
Both. I wanted to add husky to have a precommit push in the react-bootstrap project so we avoid pushing with linter errors, and also I wanted to add husky as a dev dependency in the generated project

Reply:
greatt!!!